backup_tests/xbstream_fifo_test.sh

```diff
@@ -450,6 +523,47 @@ echo "Copy the backup in datadir"
 $XTRABACKUP_DIR/bin/xtrabackup --no-defaults --copy-back --target_dir=$BACKUP_DIR/full --datadir=$DATADIR --core-file > $LOGDIR/copy_back5.log 2>&1
 start_server
 
+echo "######################################################################"
+echo "# 5. Test FIFO xbstream: Test with encrypted tables w/ keyring kmip  #"
```

Comment:
Make the testname 
6. Test FIFO xbstream: Test with encrypted tables w/ keyring kmip  #"